components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -82,13 +82,21 @@ BasePrizePool.config = {
 	storeSmw = {
 		default = true,
 		read = function(args)
-			return Logic.readBoolOrNil(args.storesmw)
+			local disabledVariable = Logic.readBoolOrNil(Variables.varDefault('disable_LPDB_storage'))
+			return Logic.nilOr(
+				Logic.readBoolOrNil(args.storesmw),
+				disabledVariable == nil and nil or not disabledVariable
```

Comment:
This won't work and the `or` will be triggered for nil values on previous :D

Reply:
I'm not sure if i understand, what will be triggered?

Reply:
`X == nil and nil or 'foo'` will always end up with `foo` set, as `nil` is falsely (and hence the `or` needs to be evaluated)